components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/inbound/oauth2/OAuthConsumerAppToApiModel.java

```diff
@@ -99,10 +100,14 @@ private IdTokenConfiguration buildIdTokenConfiguration(OAuthConsumerAppDTO oAuth
 
         return new IdTokenConfiguration()
                 .expiryInSeconds(oAuthConsumerAppDTO.getIdTokenExpiryTime())
-                .audience(getAudiences(oAuthConsumerAppDTO))
+                .audience(getIdTokenAudiences(oAuthConsumerAppDTO))
                 .encryption(buildIdTokenEncryptionConfiguration(oAuthConsumerAppDTO));
     }
 
+    /**
+     * @deprecated use {@link #getIdTokenAudiences()} instead.
+     */
+    @Deprecated
```

Comment:
deprecating private methods are not required. you can directly modify the method

Reply:
fixed with https://github.com/wso2/identity-api-server/pull/391/commits/af6b2573d8a18761b28b99b171c17f3c92248e68